frontend/src/pages/associations/events/List.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { useQuery } from "react-query";
+import { Link, useHistory } from "react-router-dom";
+import { api } from "../../../services/apiService";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import { Col } from "react-bootstrap";
+import { PageTitle } from "../../../utils/common";
+import { Tag } from "../../../utils/Tag";
+import { LoadingAssociation } from "../Loading";
+import moment from "moment";
+
+export const AssociationListEvent = ({ association, ...props }) => {
+    
+    const associationId = association.id;
+    const { data, isLoading, error } = useQuery(
+        ["events.get", { associationId }],
+        api.events.list
+    );
+    const history = useHistory();
+    
+    console.log(isLoading);
+
+    let addButton;
+    if (association.myRole.mediaPermission) {
```

Comment:
Not sure media is the right kind of permission

Reply:
Found this into the backend :
-> event_permission: true
I assume this is the persmission